src/main/java/com/example/medical_clinic_proxy/controller/VisitController.java

```diff
@@ -0,0 +1,39 @@
+package com.example.medical_clinic_proxy.controller;
+
+import com.example.medical_clinic_proxy.model.PageContent;
+import com.example.medical_clinic_proxy.model.dto.VisitDTO;
+import com.example.medical_clinic_proxy.service.VisitService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("/visits")
+@RequiredArgsConstructor
+public class VisitController {
+    private final VisitService visitService;
+
+    @GetMapping("/{patientEmail}")
+    public PageContent<VisitDTO> getVisitsByPatient(@PathVariable("patientEmail") String patientEmail, Pageable pageable) {
+        return visitService.getVisitsByPatient(patientEmail, pageable);
+    }
+
+    @PatchMapping("/{patientEmail}/{visitId}")
+    public VisitDTO reserveVisit(@PathVariable("patientEmail") String email, @PathVariable("visitId") String id) {
```

Comment:
localhost:8080/visits/kg@gmail.com/1 <- co to znacza to 1 i co oznacza ten mail 

PATCH localhost:8080/visits/reserve + w body ReserveVisitCommand